packages/tools/src/tools/StackScrollTool.ts

```diff
@@ -34,6 +36,21 @@ export default class StackScrollTool extends BaseTool {
     const targetId = this.getTargetId(viewport);
     const { invert } = this.configuration;
 
-    scrollThroughStack(viewport, targetId, deltaFrames, invert);
+    let volumeId;
+    if (viewport instanceof VolumeViewport) {
+      volumeId = targetId.split('volumeId:')[1];
+    }
+
+    // We need this check since the deltaFrames can be 0 when the user is
+    // scrolling very slowly so in that case we use the previous direction
+    let direction;
+    if (deltaFrames === 0) {
+      direction = this.previousDirection;
```

Comment:
Hmm, do we actually want to do this? Want it cause overshooting when you change direction?